### PR TITLE
Budget App (RoR)

### PR DESCRIPTION
**Pull request by pratap-panabaka**
## TASK:

Create a Ruby on Rails application that allows the user to:
- register and log in, so that the data is private to them.
- introduce new transactions associated with a category.
- see the money spent on each category.

## ACTION:
Created the Budget App which does above

## Budget App Deploymnet
Budget App is  deployed through Render website - [Open Budget App](https://budgetapp-07il.onrender.com/)

## Functionality of the App
BudgetApp

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Not applicable yet](https://loom.com)
+- [Loom Video Link](https://www.loom.com/share/3dee4d0a6097447ba72065ec0c1a5325)
```
Comment (DammyShittu) [inline README.md]: - [ ] Nice video presentation. Your video length is 6 minutes, and 8 seconds but it is required not to exceed 5 minutes. Kindly re-record your video and keep its length within 5 minutes. One of the reasons for specifying video lengths in these types of projects is to develop the ability to make presentations in a more precise and concise manner.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @PRATAP-KUMAR,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (pratap-panabaka) [reply README.md]: @DammyShittu Thank you very much for your review and the valuable point about the recording time.
I have just made another video which is just near to 5 minutes.

**Review by t-yanick — APPROVED** [review]
Hi @PRATAP-KUMAR, 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: Good job with video presentation

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**